contrib/test_jenkins.sh

```diff
@@ -868,34 +878,39 @@ test_unused_env_var() {
 
 test_env_var_aliases() {
 	echo "==== Running MLX5 env var aliases test ===="
-	vars=( "TM_ENABLE" "TM_MAX_BCOPY" "TM_LIST_SIZE" "TX_MAX_BB" )
-	for var in "${vars[@]}"
-	do
-		for tl in "RC_MLX5" "DC_MLX5"
+	if [[ `./src/tools/info/ucx_info -b | grep -P 'HW_TM *1$'` ]]
+	then
+		vars=( "TM_ENABLE" "TM_MAX_BCOPY" "TM_LIST_SIZE" "TX_MAX_BB" )
```

Comment:
`TM_MAX_BCOPY` would need to be renamed to `TM_SEG_SIZE` when #3524 is merged

Reply:
lets remove TM_MAX_BCOPY